Assignment9/index.js

```diff
@@ -0,0 +1,63 @@
+async function fetchData(){
+    const url = "https://dummyjson.com/users"
+    try {
+        const response = await fetch(url)
+        const data = await response.json()
+        return data.users
+    } catch (error) {
+        console.log("Error is: " , error);
+    }
+}
+
+
+async function addValueIntoTable(){
+    const userData = await fetchData()
+    const userTable = document.getElementById('tableBody')
+
+    userTable.innerHTML = ""
+
+    userData.forEach((user) => {
+        const row = document.createElement('tr')
+        row.innerHTML = `<td>${user.id}</td>
+                         <td>${user.firstName}</td>
+                         <td>${user.lastName}</td>
+                         <td>${user.email}</td>`;
+        userTable.appendChild(row)
+    });
+
+    document.getElementById('loading').style.display = 'none';
+}
+
+
+function sortByFirstName(){
+    const userTable = document.getElementById("tableBody")
+    const rows = Array.from(userTable.children)
+
+    rows.sort((a, b) =>{
+        const firstValue = a.children[1].textContent.toLowerCase()
+        const secondValue = b.children[1].textContent.toLowerCase()
+
+        return firstValue.localeCompare(secondValue)
```

Comment:
is it possible to sort with a generic function which will sort by key with any type?
Here if we pass key id in number type localeCompare will not work